### DISCUSSION TITLE
ssl error

### DISCUSSION THREAD

**Discussion by rey900** [mampfes/hacs_waste_collection_schedule#1396] [Q&A] [answered]
Hello all, 

i am currently trying to get the waste schedule running. but i always get an ssl error and no data. have someone the same problem and a fix maybe? If i open a browser and check the certificate it looks good for me. Name ist correct, it is valid and the CA is an external one.

HTTPSConnectionPool(host='om.kaw-hameln.de', port=443):
Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))

**Comment by 5ila5** [comment, marked as answer]
We run in this problem quite often. As far as I know (limited research) this is due to the implementation of the ssl libraries. Browsers typically use their own more resilient implementation.

An easy fix is to disable SSL verification for the ICS source by setting `verify_ssl: false`

**Reply by 5ila5** [reply]
added this to the hameln documentation in #1398

**Reply by rey900** [reply]
Thanks for your quick response - that fixed the issue
